doc/help/CHECK.md

```diff
@@ -0,0 +1,83 @@
+kdb-check(1) -- Perform internal checks
+=======================================
+
+## DESCRIPTION
+This command is used to perform checks on the key database or an Elektra plugin.  
+
+## USAGE
+`kdb check [<plugin>]`  
+
+Where the option argument, `plugin` is the plugin that a user wants to check.  
+Use `-c` to pass options to that plugin.  
+If no `plugin` argument is provided a check will be performed on the key database itself.  
+The user can also use this tool to perform write tests by passing the `-f` option (please note that this can result in configuration files being changed!).  
+Special values are returned to represent the outcome of a check.  
+
+## RETURNED VALUES
+Their are two different types of checks, a check on a plugin (by specifying the name of a plugin as an argument) or a check on the key database itself.  
+
+The outcome of a check on the key database is returned as an 8-bit pattern.  
+Each bit represents a specific outcome as described below:  
+
+ * 0:
+   No errors (no output)  
+
+ * Bit 1: 
+   Warning on open.  
+
+ * Bit 2:
+   Error on open.  
+
+ * Bit 3:
+   Warning on get.  
+
+ * Bit 4:
+   Error on get.  
+
+ * Bit 5:
+   Warning on set. (only checked when `-f` is used)  
+
+ * Bit 6:
+   Error on set (only checked when `-f` is used)  
+
+ * Bit 7:
+   Warning on close.  
+
+ * Bit 8:
+   Error on close.  
+
+So if the following number was returned `#9` the user could figure out more detail by considering the bits: `00001001`  
```

Comment:
Sorry, I am still a bit confused. Which part of that output is unique to the  kdb check  command? All of it? The numbers don't seem to match with the expected bit pattern?

Or is it just the last error,  #62  which would be  00111110  meaning error on open, warning on get, error on get, warning on set (even though I didn't use -f), error on set, but no errors on close and no warning on open, only an error.